src/test/java/org/wildfly/common/iteration/CompositeIterableTestCase.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Wrong license header

Reply:
Oops - forgot to fix the license on the test cases.  Good catch.